schemas/user.py

```
@@ -0,0 +1,128 @@
+# users의 관련된 pydantic 스키마 작성하는 부분
+from typing import List
```

Ruff (0.14.11)

`typing.List` is deprecated, use `list` instead

(UP035)